Admin/FieldDescription.php

```diff
@@ -98,6 +98,12 @@ public function getValue($object)
             $object = $this->getFieldValue($object, $parentAssociationMapping['fieldName']);
         }
 
+        // Support embedded object for mapping
+        // Ref: http://doctrine-orm.readthedocs.io/projects/doctrine-orm/en/latest/tutorials/embeddables.html
+        if (isset($this->getFieldMapping()['declaredField'])) {
```

Comment:
Hi, beware this syntax (`Function array dereferencing`) is only supported from php 5.4 (http://php.net/manual/en/migration54.new-features.php)